### ISSUE TITLE
Start up issue

### ISSUE THREAD

**Issue by beau6183** [YOU54F/serverless-offline-ses#20]
It appears that the aws-ses-local bin path is not resolving correctly.
```
Serverless: serverless-offline-ses - Starting ses with command /path/to/project/node_modules/serverless-offline-ses/node_modules/.bin/aws-ses-local --port 9001 --outputDir ./output
Serverless: serverless-offline-ses process failed to start with code 127
 
  Error --------------------------------------------------
 
  Error: Unable to start the serverless-offline-ses process
      at ServerlessOfflineSesPlugin.spawnSesProcess (/path/to/project/node_modules/serverless-offline-ses/src/index.ts:87:13)
      at Object.ServerlessOfflineSesPlugin.startSes [as hook] (/path/to/project/node_modules/serverless-offline-ses/src/index.ts:141:22)
```

**Comment by szenius** [comment]
Hi I am also facing the same issue. Is there any update on this?

**Comment by YOU54F** [comment]
sorry for delay, released as 0.0.7
